File: Resueltos_por_competidor/Jose_Caicedo/Games.cpp
/*
* Autor: BlackJose17
* Problema: Games
* Juez Online: Codeforces
* Veredicto: Accepted
* URL: https://codeforces.com/problemset/problem/268/A
**/

#include <bits/stdc++.h>
using namespace std;

int main() {
    int n;
    cin >> n;

    vector<int> h(n), a(n);
    for (int i = 0; i < n; i++) cin >> h[i] >> a[i];

    int c = 0;

    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            if (i == j) continue;
            if (h[i] == a[j]) c++;
        }
    }

    cout << c << endl;
    return 0;
}

File: Resueltos_por_competidor/Jacobo_Segura/Games.java
/* 
Codeforces 268A - Games
https://codeforces.com/problemset/problem/268/A
*/
import java.util.Scanner;;

public class Main {
    public static void main(String[] args) {
    	Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int[] h = new int[n];
        int[] g = new int[n];
        
        for(int i = 0; i<n; i++) {
        	h[i] = sc.nextInt();
        	g[i] = sc.nextInt();
        }
        
        int cont = 0;
        for(int i = 0; i<n; i++) {
        	for(int j = 0; j<n; j++) {
        		if(i != j && h[i] == g[j]) {
        			cont++;
        		}
        	}
        }
        System.out.println(cont);
    }
}
